CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+
+<a name="Messaging Go Mod Changelog"></a>
+## Messaging Module (in Go)
+[Github repository](https://github.com/edgexfoundry/go-mod-messaging)
+
+## [2.0.0] - 2021-06-30
+### Features ✨
+- Add ReceivedTopic to MessageEnvelope & remove Checksum ([#192d447](https://github.com/edgexfoundry/go-mod-messaging/commits/192d447))
+    ```
+    BREAKING CHANGE:
+    Checksum property has been removed from the MessageEnvelope
+    ```
+### Bug Fixes 🐛
+- Use Redis Pub/Sub which supports topic scheme with wild cards ([#e3da10d](https://github.com/edgexfoundry/go-mod-messaging/commits/e3da10d))
```

Comment:
This should have a breaking change note.

BREAKING CHANGE: `Redis Pub/Sub` is not compatible with previous `Redis Streams` implementation. All clients must be using the new implementation in order to properly send and receive messages.